youtube_dl/extractor/qqmusic.py

```diff
@@ -119,6 +119,12 @@ def _real_extract(self, url):
             'upload_date': publish_time,
             'creator': singer,
             'description': lrc_content,
+            'subtitles': {
+                'zh-CN': [{
```

Comment:
This is not always the case. Here is the `en` lyrics http://y.qq.com/#type=song&mid=001JyApY11tIp6.